src/layouts/MainAppLayout/Header/Header.js

```diff
@@ -0,0 +1,78 @@
+import "./Header.module.css";
```

Comment:
Organize imports, check out this article below 

https://manurana.medium.com/organizing-imports-in-react-and-react-native-faf982a3a3b5